### ISSUE TITLE
Infinite loading after selecting the account in the dialog for sign in

### ISSUE THREAD

**Issue by emilianocds** [google/google-authenticator-android#128]
Found a bug in the Android app? Thanks for reporting it! To help us out, fill out this template to the best of your ability.

Please note that the code in this repository is an open source fork of the app in the Play Store, so **there is no guarantee that both apps will be in sync**. Please check which version of the app your bug applies to below.

- [ ] This issue is present in the app downloaded from the [Play Store][playstore].
- [ ] This issue is with the [Github version][opensource] of the app.

## Describe the Bug

I added the v19.0.0 for google auth in Android and implement all the steps that are in the Google developer page.
I trigger the pop up to select an account for sign in and after that there is a pop up that has a progress bar that loads for ever. I do not get any error on the logcat

### Intended Behavior

What did you expect to happen?

### Actual Behavior

What actually happened?

## Reproducing

How does one encounter this error. If possible, provide steps that we can take, like as shown below:

1. val reg = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
        }
        reg.launch(googleSignInClient.signInIntent)
2. The pop up for selecting account is displayed
3. Click on any account
4. A new pop up is displayed with a progress bar that loads for ever and its stuck there

### Screenshots

![device-2021-01-12-165631](https://user-images.githubusercontent.com/55559063/104365757-250a0100-54f7-11eb-8658-d455138d0e1c.png)

### Device

Fill out the details about the device and app version in which you encountered the bug.

 - Device: Samsung S10+
 - OS: Android
 - Version 10

**Comment by ThomasHabets** [comment]
Please check one of the boxes in the template.

**Comment by ThomasHabets** [comment]
Actually, I'm sorry I'm not following this bug report at all. Is this even about the Google Authenticator app? You've not filled in the template so it's hard to follow.

**Comment by goldcrock** [comment]
same issue here.

how to reproduce.

1. Remove gpgs id for good in Play app.
2. execute the code


GoogleSignInOptions gso = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_GAMES_SIGN_IN)
                .requestScopes(Games.SCOPE_GAMES)
                .requestProfile()
                .requestId()
                .requestEmail()
                .build();
        GoogleSignInClient signInClient = GoogleSignIn.getClient(activity, gso);
        Intent intent = signInClient.getSignInIntent();
        activity.startActivityForResult(intent, RC_SIGN_IN);

3. account select dialog pops up.
4. select the account to sign in
5. account creation dialog pops up
6. click the create button.
7. progress bar shows up and doesn't get disappeared.

* The issue doesn't occur If scopes got eliminated when creating GoogleSignInOptions object.

* Permission request pops up If SignInIntent got called on another phone with same build.
* Permission request pops up If few hours elapsed without doing any other action
